corehq/apps/app_manager/static/app_manager/js/details/column.js

```diff
@@ -113,6 +113,12 @@ export default function (col, screen) {
             optimizationOptions,
         ).val(self.original.optimization || "");
         self.$optimizationSelectElement = $('<div/>').append(self.optimizationSelectElement.ui);
+        // Make it possible to observe changes to self.optimizationSelectElement
+        // same as done for self.field
+        self.optimizationSelectElement.observableVal = ko.observable(self.optimizationSelectElement.val());
+        self.optimizationSelectElement.on("change", function () {
+            self.optimizationSelectElement.observableVal(self.optimizationSelectElement.val());
+        });
```

Comment:
Hey @orangejenny 

I am open to suggestions to do this differently. Just checking if you have thoughts on this.

Another way I did this was to add an observable `optimization` which was then set to the optimizationSelectElement value on its 'change' and then I referred the observable optimization instead of `self.optimizationSelectElement.val())`

Though then I saw this `observableVal` hack used for `self.field` in the same file, so I used the same instead.

Reply:
Hey, I'm not going to have a chance to look at this, don't need to block on my feedback.